web/src/title/LoadSavedProject.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import { stripPrefix, stripSuffix } from "../common";
+  import { loadFromLocalStorage } from "./loader";
+
+  export let loading: string;
+
+  let fileInput: HTMLInputElement;
+
+  async function loadFile(e: Event) {
+    let filename = fileInput.files![0].name;
+    loading = `Loading from file ${filename}`;
+
+    let contents = await fileInput.files![0].text();
+    let gj = JSON.parse(contents);
+    // Is this a CNT project or a regular one?
+    let key = "";
+    if (gj.study_area_name && gj.study_area_name.startsWith("LAD_")) {
```

Comment:
This is... brittle and gross. I think part of #207 should also consider adding in appFocus or similar to the GJ savefile itself.